VideoWeb/VideoWeb.Common/Models/ConferenceVideoControlStatuses.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace VideoWeb.Common.Models
+{
+    public class ConferenceVideoControlStatuses : IComparable<ConferenceVideoControlStatuses>
+    {
+        public ConferenceVideoControlStatuses()
+        {
+            ParticipantIdToVideoControlStatusMap = new Dictionary<string, VideoControlStatus>();
+        }
+        
+        public Dictionary<string, VideoControlStatus> ParticipantIdToVideoControlStatusMap { get; set; }
+        
+        public int CompareTo(ConferenceVideoControlStatuses comparisonConferenceVideoControlStatuses)
+        {
+            bool areEqual = true;
+            foreach (var (key, videoControlStatus) in ParticipantIdToVideoControlStatusMap)
+            {
+                var comparisonvideoControlStatus = comparisonConferenceVideoControlStatuses.ParticipantIdToVideoControlStatusMap[key];
+                areEqual = areEqual && videoControlStatus.CompareTo(comparisonvideoControlStatus) > 0;
+            }
```

Comment:
This could be changed to stop comparing once one that is not equal is found.

Reply:
so do you mean to return after areEqual is false?

Reply:
Yeah that's what I was thinking. If the current .CompareTo() is 0 then return 0. And return 1 after the loop. 

This isn't a showstopper or anything.

Reply:
I think it makes sense, made the code change requested